### ISSUE TITLE
IIIF related links missing full URL

### ISSUE THREAD

**Issue by benwbrum** [benwbrum/fromthepage#1030]
In the [derivative manifests we produce](https://fromthepage.com/iiif/918/manifest), the `related` element `@id` is not a valid URL -- apparently we are leaving out the `:host => true` parameter:

```js
  "related": [
    {
      "format": "text/html",
      "label": "Read Formation of an Irish Brigade in Germany ; Manifesto of A Company of the Irish Brigade [Handwritten summary by Roger Casement of the agreement with Germany; & manifesto of the Irish Brigade, handwritten by Casement & printed by Limburger Vereinsdruckerei]",
      "@id": "/jbhoward/boehm-casement-papers/formation-of-an-irish-brigade-in-germany-manifesto-of-a-company-of-the-irish-brigade-handwritten-summary-by-roger-casement-of-the-agreement-with-germany-manifesto-of-the-irish-brigade-handwritten-by-casement-printed-by-limburger-vereinsdruckerei"
    },
    {
      "format": "text/html",
      "label": "Table of Contents for Formation of an Irish Brigade in Germany ; Manifesto of A Company of the Irish Brigade [Handwritten summary by Roger Casement of the agreement with Germany; & manifesto of the Irish Brigade, handwritten by Casement & printed by Limburger Vereinsdruckerei]",
      "@id": "/jbhoward/boehm-casement-papers/formation-of-an-irish-brigade-in-germany-manifesto-of-a-company-of-the-irish-brigade-handwritten-summary-by-roger-casement-of-the-agreement-with-germany-manifesto-of-the-irish-brigade-handwritten-by-casement-printed-by-limburger-vereinsdruckerei/contents"
    }
  ],
```

**Comment by benwbrum** [comment]
This may already have been fixed by PR #1125.  @diegoviola, can you test this and close the issue if it's still a problem?

**Comment by diegoviola** [comment]
@benwbrum I tested this and it looks like it has already been addressed by the previous PR.
